Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp

```diff
@@ -137,7 +137,9 @@ ScrollingStateScrollingNode::ScrollingStateScrollingNode(const ScrollingStateScr
 
 ScrollingStateScrollingNode::~ScrollingStateScrollingNode()
 {
-    scrollingStateTree().scrollingNodeRemoved();
+    // handle the case when destructor being called before being attached to a tree
```

Comment:
This comment doesn't really add much of an explanation above just reading the if statement.  Could you remove it before merging the PR?

Reply:
Sure! Updated